iroha/src/account.rs

```diff
@@ -72,10 +72,17 @@ pub mod isi {
             world_state_view: &WorldStateView,
         ) -> Result<WorldStateView> {
             let mut world_state_view = world_state_view.clone();
+            let max_entry_byte_size = world_state_view.config.asset_max_entry_byte_size;
+            let max_store_len = world_state_view.config.asset_max_store_len;
             let account = world_state_view
                 .account(&self.object_id)
                 .ok_or_else(|| error!("Failed to find account."))?;
-            let _ = account.metadata.insert(self.key, self.value);
+            let _ = account.metadata.insert_with_limits(
+                self.key,
+                self.value,
+                max_entry_byte_size,
+                max_store_len,
+            );
```

Comment:
Api seems kinda clumsy. What do you think about introducing type `MetadataLimits`?